SteamKit2/SteamKit2/Steam/Handlers/SteamApps/SteamApps.cs

```diff
@@ -185,6 +185,7 @@ public AsyncJob<PICSChangesCallback> PICSGetChangesSince( uint lastChangeNumber
         /// <param name="package">Package id requested.</param>
         /// <param name="metaDataOnly">Whether to send only meta data.</param>
         /// <returns>The Job ID of the request. This can be used to find the appropriate <see cref="PICSProductInfoCallback"/>.</returns>
+        [Obsolete( "Use an overload that takes PICSRequest instead of uint. In SteamKit 2.3, this overload had four parameters, but onlyPublic was removed. Audit your code." )]
```

Comment:
I *still* believe we should just remove this overload entirely right away, so nobody disregards the warning and end up with broken functionality, but if we really intend to support it then obsoletion mark is still far better than silent breaking change.